test/unit-test/ota_utest.c

```diff
@@ -576,6 +564,19 @@ static void otaAppBufferDefault()
     pOtaAppBuffer.authSchemeSize = OTA_AUTH_SCHEME_SIZE;
 }
 
+static void processEntireQueue()
```

Comment:
Maybe add a small comment to explain the need for this function?

Reply:
This is used in a couple of places with different purposes so there isn't really one need for this function. The purpose of the function is to process all remaining items in the queue. I'll add a brief comment describing this to add clarity.